<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDetailDia</class>
 <widget class="QDialog" name="HistoryDetailDia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="thumbnailLb">
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>800</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>11</number>
         </property>
         <property name="topMargin">
          <number>11</number>
         </property>
         <item row="0" column="0">
          <widget class="Line" name="line">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>800</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QFrame" name="frame_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="1">
             <widget class="QPushButton" name="closeButton">
              <property name="font">
               <font>
                <family>Arial</family>
               </font>
              </property>
              <property name="text">
               <string>ok</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>16</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="styleSheet">
            <string notr="true">QTabBar::tab {
color: rgb(134, 134, 134);
 border: 0px ;
	border-top-color: #ffffff;
border-left-color: #ffffff;
border-right-color: #ffffff;
  border-bottom-color: #5cbcca;  
	background-color: rgb(255, 255, 255);
  
    min-width: 40ex;
    padding: 12px;
	font: 8pt &quot;Arial&quot;;

	/*margin-bottom:10px;*/
}

QTabBar::tab:selected {
    /*border-color: #9B9B9B;*/
	color: rgb(92, 188, 202);
 	border: 2px solid ;
	border-top-color: #ffffff;
border-left-color: #ffffff;
border-right-color: #ffffff;
    border-bottom-color: #5cbcca;  
 
}
QTabWidget::pane {

border: 0px solid #C2C7CB;

}</string>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Slice Setting</string>
            </attribute>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="geometry">
              <rect>
               <x>-1</x>
               <y>-4</y>
               <width>571</width>
               <height>771</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"> QScrollBar:vertical {
     border: 0px solid grey;
     background: #999999;
     width: 4px;
     margin: 0px 0 0px 0;
 }
QScrollBar::handle:vertical {  
	
	background-color: #5cbcca;
    min-width: 20px;
 	border-top-left-radius: 4px;
    border-top-right-radius: 4px;
	border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
	
}
QScrollBar::add-line:vertical {
    border: none;
      background: none;
      color: none;
}

 QScrollBar::sub-line:vertical {
    border: none;
      background: none;
      color: none;
 }
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>571</width>
                <height>771</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background: #ededed;</string>
              </property>
             </widget>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Printing Detail</string>
            </attribute>
            <widget class="QScrollArea" name="scrollArea">
             <property name="geometry">
              <rect>
               <x>-1</x>
               <y>-4</y>
               <width>571</width>
               <height>761</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"> QScrollBar:vertical {
     border: 0px solid ;
     background: #999999;
     width: 4px;
     margin: 0px 0 0px 0;
 }
QScrollBar {
     border: 0px solid ;
    
 }
QScrollBar::handle:vertical {  
	
	background-color: #5cbcca;
    min-width: 20px;
 	border-top-left-radius: 4px;
    border-top-right-radius: 4px;
	border-bottom-left-radius: 4px;
    border-bottom-right-radius: 4px;
	
}
QScrollBar::add-line:vertical {
    border: none;
      background: none;
      color: none;
}

 QScrollBar::sub-line:vertical {
    border: none;
      background: none;
      color: none;
 }
</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>571</width>
                <height>761</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">background: #ededed;</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="4" column="0">
                <widget class="QGroupBox" name="wiperGB">
                 <property name="styleSheet">
                  <string notr="true">QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
	color: rgb(134, 134, 134);
}</string>
                 </property>
                 <property name="title">
                  <string>Wiper</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QGroupBox" name="binderGB">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>112</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
	color: rgb(134, 134, 134);
}</string>
                 </property>
                 <property name="title">
                  <string>Binder</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QGroupBox" name="printheadGB">
                 <property name="styleSheet">
                  <string notr="true">QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
	color: rgb(134, 134, 134);
}</string>
                 </property>
                 <property name="title">
                  <string>Printhead</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QGroupBox" name="magentaGB">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>112</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
	color: rgb(134, 134, 134);
}</string>
                 </property>
                 <property name="title">
                  <string>Magenta</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QGroupBox" name="yellowGB">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>112</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
	color: rgb(134, 134, 134);
}</string>
                 </property>
                 <property name="title">
                  <string>Yellow</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QGroupBox" name="cyanGB">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>112</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QGroupBox::title {
    subcontrol-origin: margin;
    left: 7px;
    padding: 0px 5px 0px 5px;
	font: 8pt &quot;Arial&quot;;
	color:rgb(134, 134, 134);
}
</string>
                 </property>
                 <property name="title">
                  <string>Cyan</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
        <zorder>tabWidget</zorder>
        <zorder>frame_6</zorder>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>400</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>frame</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>frame_3</zorder>
     <zorder>frame_4</zorder>
     <zorder>frame_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
